x/profiles/keeper/migrations.go

```diff
@@ -32,7 +30,12 @@ func (m Migrator) Migrate4to5(ctx sdk.Context) error {
 	return v4.MigrateStore(ctx, m.ak, m.keeper.storeKey, m.keeper.legacyAmino, m.keeper.cdc)
 }
 
-// Migrate5To6 migrates from version 5 to 6.
-func (m Migrator) Migrate5To6(ctx sdk.Context) error {
+// Migrate5to6 migrates from version 5 to 6.
+func (m Migrator) Migrate5to6(ctx sdk.Context) error {
 	return v5.MigrateStore(ctx, m.keeper.storeKey, m.keeper.cdc, m.keeper.legacyAmino)
 }
+
+// Migrate6to7 migrates from version 6 to 7.
+func (m Migrator) Migrate6to7(ctx sdk.Context) error {
```

Comment:
I am confused the version, `v6.MigrateStore` is actually migrating to v6 from v5 although the method is called `Migrate6to7`. Is that consensus version different from legacy store version?

Reply:
That was actually an error within comments themselves. I've now updated them to make it more clear. Generally, the `Migrate<X>to<X+1>` calls the `v<X>` package that handles the migration from `X` to `X+1` properly. So in this case `Migrate6to7` calls the `v6` package that migrates from `6` to `7` (latest version). 

The only thing wrong here was the comment on the `v6.MigrateStore` function. It migrates from `6` to `7` (and not `5` to `6` as it was written). I've now fixed it

Reply:
@RiccardoM Thanks for the explanation. But it would be another problem that `v6.MigrateStore` actually migrates `v5types` to the current type `v6` [like this](https://github.com/desmos-labs/desmos/blob/628932d7360e3552fc6ee0f6a37a909f0e4070e4/x/profiles/legacy/v6/store.go#L50).